mobile/lib/main.dart

```diff
@@ -221,7 +228,34 @@ class _TenTenOneAppState extends State<TenTenOneApp> {
   void initState() {
     super.initState();
 
-    init(context.read<bridge.Config>());
+    final config = context.read<bridge.Config>();
+
+    init(config);
+
+    WidgetsBinding.instance.addPostFrameCallback((_) async {
+      PackageInfo packageInfo = await PackageInfo.fromPlatform();
+
+      final messenger = scaffoldMessengerKey.currentState!;
+
+      try {
+        final response = await http.get(
+          Uri.parse('http://${config.host}:${config.httpPort}/api/version'),
+        );
+
+        final clientVersion = Version.parse(packageInfo.version);
+        final coordinatorVersion = Version.parse(jsonDecode(response.body));
+        FLog.info(text: "Coordinator version: ${coordinatorVersion.toString()}");
+
+        if (coordinatorVersion > clientVersion) {
```

Comment:
🤔 will that work for something like "1.0.21" and "1.1.0" ?

Reply:
This is using a SemVer parsing library, and I believe 1.0.21 should compare as less than 1.1.0 in SemVer